src/redux/pattern/tests/pattern.mocks.js

```diff
@@ -0,0 +1,108 @@
+export const pattern = {
```

Comment:
Please, rename the file to the `pattern.variables`, we usually use this kind of name

Reply:
fixed